filer/static/filer/js/libs/dropzone.min.js

```diff
@@ -1,86 +1,31 @@
allelChunkUploads:!1,retryChunks:!1,retryChunksLimit:3,maxFilesize:256,paramName:"file",createImageThumbnails:!0,maxThumbnailFilesize:10,thumbnailWidth:120,thumbnailHeight:120,thumbnailMethod:"crop",resizeWidth:null,resizeHeight:null,resizeMimeType:null,resizeQuality:0.8,resizeMethod:"contain",filesizeBase:1e3,maxFiles:null,headers:null,clickable:!0,ignoreHiddenFiles:!0,acceptedFiles:null,acceptedMimeTypes:null,autoProcessQueue:!0,autoQueue:!0,addRemoveLinks:!1,previewsContainer:null,hiddenInputContainer:"body",capture:null,renameFilename:null,renameFile:null,forceFallback:!1,dictDefaultMessage:"Drop files here to upload",dictFallbackMessage:"Your browser does not support drag'n'drop file uploads.",dictFallbackText:"Please use the fallback form below to upload your files like in the olden days.",dictFileTooBig:"File is too big ({{filesize}}MiB). Max filesize: {{maxFilesize}}MiB.",dictInvalidFileType:"You can't upload files of this type.",dictResponseError:"Server responded with {{statusCode}} code.",dictCancelUpload:"Cancel upload",dictUploadCanceled:"Upload canceled.",dictCancelUploadConfirmation:"Are you sure you want to cancel this upload?",dictRemoveFile:"Remove file",dictRemoveFileConfirmation:null,dictMaxFilesExceeded:"You can not upload any more files.",dictFileSizeUnits:{tb:"TB",gb:"GB",mb:"MB",kb:"KB",b:"b"},init(){},params(t,a,l){if(l)return{dzuuid:l.file.upload.uuid,dzchunkindex:l.index,dztotalfilesize:l.file.size,dzchunksize:this.options.chunkSize,dztotalchunkcount:l.file.upload.totalChunkCount,dzchunkbyteoffset:l.index*this.options.chunkSize}},accept(t,a){return a()},chunksUploaded:function(t,a){a()},fallback(){let t;this.element.className=`${this.element.className} dz-browser-not-supported`;for(let l of this.element.getElementsByTagName("div"))if(/(^| )dz-message($| )/.test(l.className)){t=l,l.className="dz-message";break}t||(t=Dropzone.createElement("<div class=\"dz-message\"><span></span></div>"),this.element.appendChild(t));let a=t.getElementsByTagName("span")[0];return a&&(null==a.textContent?null!=a.innerText&&(a.innerText=this.options.dictFallbackMessage):a.textContent=this.options.dictFallbackMessage),this.element.appendChild(this.getFallbackForm())},resize(t,a,l,s){let d={srcX:0,srcY:0,srcWidth:t.width,srcHeight:t.height},p=t.width/t.height;null==a&&null==l?(a=d.srcWidth,l=d.srcHeight):null==a?a=l*p:null==l&&(l=a/p),a=Math.min(a,d.srcWidth),l=Math.min(l,d.srcHeight);let u=a/l;if(d.srcWidth>a||d.srcHeight>l)if("crop"===s)p>u?(d.srcHeight=t.height,d.srcWidth=d.srcHeight*u):(d.srcWidth=t.width,d.srcHeight=d.srcWidth/u);else if("contain"===s)p>u?l=a/p:a=l*p;else throw new Error(`Unknown resizeMethod '${s}'`);return d.srcX=(t.width-d.srcWidth)/2,d.srcY=(t.height-d.srcHeight)/2,d.trgWidth=a,d.trgHeight=l,d},transformFile(t,a){return(this.options.resizeWidth||this.options.resizeHeight)&&t.type.match(/image.*/)?this.resizeImage(t,this.options.resizeWidth,this.options.resizeHeight,this.options.resizeMethod,a):a(t)},previewTemplate:`\
```

Comment:
this is es6 so it won't work in slightly older browsers, also there's no mention about the version of the dropzone.js anymore which should still be there